lib/node/nodes/interpolation.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'enrich';
+
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    target: Node.PARAM.NODE(Node.GEOMETRY.ANY),
+    column: Node.PARAM.STRING(),
+    method: Node.PARAM.NULLABLE(Node.PARAM.NUMBER()),
+    p1: Node.PARAM.NULLABLE(Node.PARAM.NUMBER()),
+    p2: Node.PARAM.NULLABLE(Node.PARAM.NUMBER())
```

Comment:
Same thing here, if `p2` refers to `decay order`. Could we use `decayOrder` as param name?